R/Set_up_subfunctions.R

```diff
@@ -0,0 +1,338 @@
+#' @title
+#' Calculate key age parameters to inform life tables (for pig population)
+#' @description
+#' Calculate age rate and age widths for pig compartments depending on number of age groups 
+#'
+#' @param number_age_classes the number of age classes (pigs)
+#' @param pig_age_class_width vector for mean no. of months in each age compartment 
+#'
+#' @return age rate and age width
+#' @export
+age_parameters_pig_func <- function(number_age_classes, pig_age_class_width){
+  
+  number_age_classes_pig = number_age_classes
```

Comment:
Lines like this are defunct. Either refer to `number_age_classes` in the function body, or rename the argument `number_age_classes_pig`

Reply:
Agree, will remove unnecessary arguments like this